### DISCUSSION TITLE
How can we become org members?

### DISCUSSION THREAD

**Discussion by FunctionalMetatable** [theforumhelpers/theforumhelpers.github.io#213] [Q&A] [answered]
As of right now, only Scratchers who are a part of [The Forum Helpers](https://scratch.mit.edu/studios/3688309/) can join this organisation. However, you are still able to open issues and create pull requests even if you aren't a Forum Helper.

_Originally posted by @Accio1 in https://github.com/theforumhelpers/theforumhelpers.github.io/discussions/139#discussioncomment-624649_

So, as me, @gosoccerboy5, @BarelySmooth, @Shluffy and more are members of the studio and contributing regularly, could you guys give org invites to the people who are actively contributing to this repository, and also TFH members?

**Comment by Shluffy** [comment]
even so, most of us only have commited a few times

**Comment by leahcimto** [comment]
Simply request (as you have done here). @gosoccerboy5 is an org member, and @Accio1 can invite you all.

**Comment by Accio1** [comment, marked as answer]
If you would like to be invited and you have made contributions to the site, please leave a comment on my Scratch profile asking to be invited along with your GitHub username.

**Comment by aidan-crockett** [comment]
I'm already an organization member (happily), but it's shown as private. I do think you deserve to be an organization member @FunctionalMetatable
